File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreModelsSTD/metaschema/PrincipioAttivo.cs
using AgronicaCoreModelsSTD.baseClass;
using System;
using System.Collections.Generic;
using System.Text;

namespace AgronicaCoreModelsSTD.metaschema
{
    public class PrincipioAttivo: BaseCodeDescr
    {
        public decimal titolo { get; set; }
        public decimal peso { get; set; }

        public decimal percentualeSuperficieTrattabile { get; set; }

        public PrincipioAttivo(int codice) : base(codice, "")
        {
        }
        public PrincipioAttivo() : base() { }
    }
}

File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/metaschema/PrincipioAttivo.ts
import { BaseCodeDescr } from '../baseClass/baseCodeDescr';

export class PrincipioAttivo extends BaseCodeDescr {

    titolo: number;
    peso: number;
    percentualeSuperficieTrattabile: number;

    constructor(codice: number) {
        super(codice);
    }
}
